fastly/logentries_test.go

```diff
@@ -126,6 +130,7 @@ func TestClient_Logentries(t *testing.T) {
 			Name:           "test-logentries",
 			NewName:        String("new-test-logentries"),
 			FormatVersion:  Uint(2),
+			Region:         String("ap"),
```

Comment:
Do we not need an assertion to validate the update of Region happened successfully?

Reply:
Good call. I added that check.